python-lib/design/split_format.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+
+def add_group_column(input_df, A_list, B_list, column_name, attribution_method):
```

Comment:
`attribution_method` is not used inside the function

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/1/commits/a2c91e1042972154cfc9799db57d51536fb5463c